src/context/product-context.js

```diff
@@ -0,0 +1,52 @@
+import { createContext,useContext, useEffect,useReducer  } from "react";
+import axios from "axios";
+import { sortReducer } from "../reducer/product-filter-reducer";
```

Comment:
You can export this from a single file just like previous comment of mine as you will be making more reducers later on

Reply:
I will do that. Thanks